2/Dockerfile

```diff
@@ -17,7 +17,9 @@ MAINTAINER Ben Parees <bparees@redhat.com>
 ENV JENKINS_VERSION=2.46.1 \
     HOME=/var/lib/jenkins \
     JENKINS_HOME=/var/lib/jenkins \
-    JENKINS_UC=https://updates.jenkins-ci.org
+    JENKINS_UC=https://updates.jenkins-ci.org \
+    LANG en_US.UTF-8 \
```

Comment:
The image's s2i test actually caught this ... you need an `=` between `LANG` and `en_US.UTF-8`, same for `LC_ALL`.

The msg from the test was `The s2i test failed with `Error response from daemon: Syntax error - can't find = in "LANG". Must be of the form: name=value``